deploy/examples/create-external-cluster-resources.py

```diff
@@ -32,6 +32,7 @@
 if sys.version_info.major >= 3:
     py3k = True
     import urllib.parse
+    from ipaddress import ip_address, IPv4Address
```

Comment:
also, if this module is not there by default, we may need to document or mention

Reply:
No it is the library